src/OpenTelemetry.Contrib.Instrumentation.GrpcCore/OpenTelemetry.Contrib.Instrumentation.GrpcCore.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>.NET gRPC Core based client and server interceptors for OpenTelemetry.</Description>
+    <PackageTags>$(PackageTags);OpenTelemetry;gRPC Core;interceptors</PackageTags>
+    <MinVerTagPrefix>Instrumentation.GrpcCore-</MinVerTagPrefix>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="[3.6.1,4.0)" />
+    <PackageReference Include="Grpc" Version="[2.23.0,3.0)" />
+    <PackageReference Include="OpenTelemetry" Version="1.0.1" />

```

Comment:
You should be good to just add OpenTelemetry.API here, right?

Reply:
yeah I think so.